src/__experimental__/components/label/label.component.js

```diff
@@ -10,14 +12,27 @@ const Label = ({
   helpId,
   helpTag,
   helpTabIndex,
+  hasError,
+  hasWarning,
+  hasInfo,
+  tooltipMessage,
   ...props
 }) => (
   <LabelStyle
     data-element='label'
+    hasError={ hasError }
     { ...props }
   >
     {children}
-    {help && (
+
+    {(hasError || hasWarning || hasInfo) && tooltipMessage && (
```

Comment:
Might be nice to have this in a helper. E.g.

`const validationsPresent = ({ hasError, hasWarning, hasInfo }) => hasError || hasWarning || hasInfo;`